css/style.css

```diff
@@ -0,0 +1,2896 @@
+    background-size: 450px 100px;
+    background-position: 20px 100px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: 0 150px;
+  }
+
+}
+
+@media (max-width: 500px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .top {
+    padding: 90px 0 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 160px;
+    background-repeat: no-repeat;
+    background-size: 500px 271px;
+  }
+
+  .top__wrapper {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 20px;
+  }
+
+  .top__wrapper-charset {
+    display: flex;
+    align-items: center;
+    margin-left: 50px;
+    margin-right: 0;
+    padding-right: 0;
+    background: url(../img/svg/road-mobile.svg) no-repeat;
+    background-position: 210px 5px;
+  }
+
+  .top__wrapper-item {
+    display: flex;
+    justify-content: center;
+    margin-top: 15px;
+    margin-bottom: 25px;
+    margin-right: 25px;
+    margin-left: 0;
+  }
+
+  .charset {
+    margin: 0 10px;
+  }
+
+  .charset__item {
+    margin-right: 0;
+    margin-left: 20px;
+  }
+
+  .charset__first {
+    margin: 0 2px 0 0;
+  }
+
+  .charset__speed {
+    width: 38px;
+    height: 28px;
+    margin-left: 26px;
+    margin-bottom: 12px;
+    margin-right: 10px;
+  }
+
+  .charset__road {
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .charset__first-big {
+    font-weight: 600;
+    font-size: 27px;
+    line-height: 33px;
+    padding-left: 0;
+  }
+
+  .charset__first-small {
+    font-size: 15px;
+    padding-bottom: 0;
+    padding-right: 21px;
+    margin-right: 27px;
+  }
+
+  .charset__second{
+    font-size: 10px;
+    padding-left: 0;
+  }
+
+  .button__outline {
+    color: #e51937;
+    padding-right: 20px;
+    padding-left: 20px;
+    padding-top: 13px;
+    background-position: 0 10px;
+    background-size: 22px 22px;
+  }
+
+  .button__outline-img {
+    margin-left: 5px;
+  }
+
+  .button__red {
+    padding-left: 25px;
+    padding-right: 25px;
+    padding-bottom: 8px;
+    padding-top: 8px;
+    margin-right: 0;
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .button__text {
+    margin-right: 0;
+    padding-right: 0;
+    padding-left: 0;
+    font-size: 13px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 400px 100px;
+    background-position: 20px 100px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: 0 130px;
+  }
+ 
+}
+
+@media (max-width: 450px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .header__logo {
+    padding-left: 0;
+  }
+
+  .header__right {
+    margin-right: 20px;
+  }
+
+  .top {
+    padding: 90px 0 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 160px;
+    background-repeat: no-repeat;
+    background-size: 450px 271px;
+  }
+
+  .top__wrapper { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)